templates/components/styles/global_nav.html.twig

```diff
@@ -28,7 +28,7 @@
                             {% if navItem.children is defined and navItem.children|length > 0 %}
                                 <ul class="clean-list" role="list">
                                     {% for child in navItem.children %}
-                                        <li>
+                                        <li {{ child.startNewColumn ? 'class="break-after"' : '' }}>
```

Comment:
```suggestion
                                        <li {{ (child.startNewColumn is defined and child.startNewColumn) ? 'class="break-after"' : '' }}>
```
This should allow merging this PR without having to deploy other projects first

Reply:
Yes, of course! Why didn't I think of that? :)
I've made the change and tested locally when startNewColumn is not defined.
Ready to be merged!
Thanks @jean-gui